content/medo_manual.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Medo
+sidebar: false
+---
+
+Medo is een mini-Elektronisch Patiënten Dossier dat speciaal is gebouwd om
```

Comment:
Willen we Medo nog wel presenteren als een EPD? Is het voor de onderzoekers niet een `Systeem om gegevens over deelnemers bij te houden`?